presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatisticType.java

```diff
@@ -21,7 +21,8 @@ public enum ColumnStatisticType
     NUMBER_OF_DISTINCT_VALUES("approx_distinct"),
     NUMBER_OF_NON_NULL_VALUES("count"),
     NUMBER_OF_TRUE_VALUES("count_if"),
-    TOTAL_SIZE_IN_BYTES("sum_data_size_for_stats");
+    TOTAL_SIZE_IN_BYTES("sum_data_size_for_stats"),
+    HISTOGRAM("tdigest_agg");
```

Comment:
I've put this here for now as it's the only available function currently that can generate a histogram in Presto. This may be updated in a follow-up PR to something like KLL

Reply:
qdigest_agg should also be able to do this.

Reply:
Ah you're right. Currently, tdigest  is just the default value since one is required. Connectors can (need?) to specify the function used for the histogram statistic anyways since the metastore(s) might support only some formats (e.g. Hive only supporting KLL). I don't think the default here matters all that much